cmd/kubeadm/app/api/env.go

```diff
@@ -0,0 +1,33 @@
+package api
+
+import (
+	"fmt"
+	"os"
+	"runtime"
+	"strings"
+)
+
+// TODO(phase2) use componentconfig
```

Comment:
I don't think this is compelte until we have a real apigroup. I'll remove once that's done.